tests/should-pass/iati-activities/iati-activity/default-aid-type/_vocabulary/attribute-exists.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity>
+    <iati-identifier></iati-identifier>
+    <reporting-org type="xx" ref="xx"><narrative>Organisation name</narrative></reporting-org>
```

Comment:
For info, I've added an issue to fix the formatting of this line en masse before release: https://github.com/IATI/IATI-Schemas/issues/403